DM.MovieApi.IntegrationTests/ApiResponseUtil.cs

```diff
@@ -268,5 +268,75 @@ private static void AssertGenres( IReadOnlyList<int> genreIds, IReadOnlyList<Gen
                 Assert.IsTrue( genre.Id > 0 );
             }
         }
+
+        public static void AssertTVShowSeasonInformationStructure(SeasonInfo seasonInfo)
+        {
+            // ReSharper disable once PossibleMultipleEnumeration
+            Assert.IsTrue(seasonInfo.Episodes.Any());
+
+            // ReSharper disable once PossibleMultipleEnumeration
+            foreach (var episode in seasonInfo.Episodes)
+            {
+                AssertTVShowSeasonInformationStructure(episode);
+            }
+        }
+
+        private static void AssertTVShowSeasonInformationStructure(Episode episode)
+        {
+            Assert.IsTrue(episode.Id > 0);
+            Assert.IsFalse(episode.AirDate == default(DateTime));
```

Comment:
@mejackhumphrey The `(DateTime)` type can be removed, re: `(episode.AirDate == default);`